examples/internal/proto/examplepb/response_body_service.swagger.json

```diff
@@ -44,6 +44,45 @@
         ]
       }
     },
+    "/responsebody/stream/{data}": {
+      "get": {
+        "operationId": "ResponseBodyService_GetResponseBodyStream",
+        "responses": {
+          "200": {
+            "description": "(streaming responses)",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "$ref": "#/definitions/examplepbResponseBodyOutResponse"
```

Comment:
This definition is not correct now, it should point to a `string` field with name `data`.

Reply:
It is auto-generated file. Should it require any other changes to correct it ?

Reply:
Yeah, you will need to have a look at the logic in the swagger generator to make sure it uses the ResponseBody type when it is specified in the proto options. This may already be done for the unary case, so might not be too bad to port. See https://github.com/grpc-ecosystem/grpc-gateway/blob/6b0fe46fad37b6eef2b5c32ded5aed9af08911d6/protoc-gen-swagger/genswagger/template.go#L912-L919

Reply:
i checked the swagger json, it looks good to me.
```
"responses": {
          "200": {
            "description": "(streaming responses)",
            "schema": {
              "type": "object",
              "properties": {
                "result": {
                  "$ref": "#/definitions/examplepbResponseBodyOutResponse"
                },
                "error": {
                  "$ref": "#/definitions/runtimeStreamError"
                }
              },
              "title": "Stream result of examplepbResponseBodyOut"
            }
          },
          "default": {
            "description": "An unexpected error response",
            "schema": {
              "$ref": "#/definitions/runtimeError"
            }
          }
        }

"examplepbResponseBodyOutResponse": {
      "type": "object",
      "properties": {
        "data": {
          "type": "string"
        }
      }
    }
```
without response body:
```
"responses": {
          "200": {
            "description": "A successful response.(streaming responses)",
            "schema": {
              "type": "object",
              "properties": {
                "result": {
                  "$ref": "#/definitions/examplepbResponseBodyOut"
                },
                "error": {
                  "$ref": "#/definitions/runtimeStreamError"
                }
              },
              "title": "Stream result of examplepbResponseBodyOut"
            }
          },
          "default": {
            "description": "An unexpected error response",
            "schema": {
              "$ref": "#/definitions/runtimeError"
            }
          }
        },


"examplepbResponseBodyOut": {
      "type": "object",
      "properties": {
        "response": {
          "$ref": "#/definitions/examplepbResponseBodyOutResponse"
        }
      }
    }
```

Reply:
Nice, you're right! Sorry about that.